tests/test_valuerep.py

```diff
@@ -907,6 +908,7 @@ def test_float_value(self):
         with pytest.raises(TypeError):
             _ = IS(14.5, validation_mode=config.RAISE)
 
+    @pytest.mark.filterwarnings("ignore:Value")
```

Comment:
I don't think we should filter these warnings - they should not be issued with the config flags RAISE in one and IGNORE in the other.  I think the underlying code should be fixed instead. (but we can do that in a separate PR).

Reply:
Agreed. Fixed in 3d0f916.